libraries/AP_Arming/AP_Arming.h

```diff
@@ -68,6 +67,14 @@ class AP_Arming {
     // get expected magnetic field strength
     uint16_t compass_magfield_expected() const;
 
+    // rudder arming support
+    enum ArmingRudder {
+        ARMING_RUDDER_DISABLED  = 0,
+        ARMING_RUDDER_ARMONLY   = 1,
+        ARMING_RUDDER_ARMDISARM = 2
+    };
+    ArmingRudder rudder_arming() const { return (ArmingRudder)_rudder_arming.get(); }
```

Comment:
Maybe call the function `get_rudder_arming_type(void)`? The function name of `rudder_arming()` isn't very clear from an outside caller what is necessarily happening.